decoder/vaapidecoder_base.cpp

```diff
@@ -346,6 +346,16 @@ Decode_Status
         INFO("setting LIBVA_DRIVER_NAME to wrapper for chromeos");
     }
 #endif
+
+    if (profile == VAProfileVP9Version0) {
```

Comment:
seems not good.Nobody clean the env, it cause problem if you play vp9 and then want play vp8.  And if you want them play at same time, it will have issue too.We need find better way.